From 4366ac17628a57848d509473bc19c8f5dced7005 Mon Sep 17 00:00:00 2001
From: Pierre-Hugues Husson <phh@phh.me>
Date: Tue, 16 Jul 2024 12:48:43 +0000
Subject: [PATCH 1/2] Disable vndklite handling

---
 linker/linker.cpp | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/linker/linker.cpp b/linker/linker.cpp
index a5913c26d..2ef56d2bc 100644
--- a/linker/linker.cpp
+++ b/linker/linker.cpp
@@ -95,11 +95,10 @@ static uint64_t g_module_load_counter = 0;
 static uint64_t g_module_unload_counter = 0;
 
 static const char* const kLdConfigArchFilePath = "/system/etc/ld.config." ABI_STRING ".txt";
 
 static const char* const kLdConfigFilePath = "/system/etc/ld.config.txt";
-static const char* const kLdConfigVndkLiteFilePath = "/system/etc/ld.config.vndk_lite.txt";
 
 static const char* const kLdGeneratedConfigFilePath = "/linkerconfig/ld.config.txt";
 
 #if defined(__LP64__)
 static const char* const kSystemLibDir        = "/system/lib64";
@@ -3538,14 +3537,10 @@ static std::string get_ld_config_file_apex_path(const char* executable_path) {
   }
   return "";
 }
 
 static std::string get_ld_config_file_vndk_path() {
-  if (android::base::GetBoolProperty("ro.vndk.lite", false)) {
-    return kLdConfigVndkLiteFilePath;
-  }
-
   std::string ld_config_file_vndk = kLdConfigFilePath;
   size_t insert_pos = ld_config_file_vndk.find_last_of('.');
   if (insert_pos == std::string::npos) {
     insert_pos = ld_config_file_vndk.length();
   }
-- 
2.45.1
