From 086e3ebc8ab512af67437d2676cfe304c0208951 Mon Sep 17 00:00:00 2001
From: "kaii-lb" <strawberry>
Date: Thu, 14 Mar 2024 19:58:26 +0000
Subject: [PATCH 1/1] add vndk v28 apex

---
 Android.bp | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/apex/Android.bp b/apex/Android.bp
index 3e20230..7668288 100644
--- a/apex/Android.bp
+++ b/apex/Android.bp
@@ -89,3 +89,11 @@ apex_vndk {
     system_ext_specific: true,
 }
 
+apex_vndk {
+    name: "com.android.vndk.v28",
+    defaults: ["vndk-apex-defaults"],
+    vndk_version: "28",
+    system_ext_specific: true,
+}
+
+
