From bff98f2770714f469d9b5b70d588cba8bb48bac7 Mon Sep 17 00:00:00 2001
From: kaii-lb <aosp@crave.io>
Date: Sat, 3 Aug 2024 23:21:19 +0000
Subject: [PATCH] Disable onInputDeviceChanged to solve soft bootloop

---
 .../input/KeyboardBacklightController.java    | 34 ++++++++++---------
 1 file changed, 18 insertions(+), 16 deletions(-)

diff --git a/services/core/java/com/android/server/input/KeyboardBacklightController.java b/services/core/java/com/android/server/input/KeyboardBacklightController.java
index c3205afe14f2..aa561464e3b4 100644
--- a/services/core/java/com/android/server/input/KeyboardBacklightController.java
+++ b/services/core/java/com/android/server/input/KeyboardBacklightController.java
@@ -374,26 +374,28 @@ final class KeyboardBacklightController implements
     }
 
     @VisibleForTesting
     @Override
     public void onInputDeviceChanged(int deviceId) {
-        InputDevice inputDevice = getInputDevice(deviceId);
-        if (inputDevice == null) {
-            return;
-        }
-        final Light keyboardBacklight = getKeyboardBacklight(inputDevice);
-        if (keyboardBacklight == null) {
-            mKeyboardBacklights.remove(deviceId);
-            return;
-        }
-        KeyboardBacklightState state = mKeyboardBacklights.get(deviceId);
-        if (state != null && state.mLight.getId() == keyboardBacklight.getId()) {
-            return;
-        }
-        // The keyboard backlight was added or changed.
-        mKeyboardBacklights.put(deviceId, new KeyboardBacklightState(deviceId, keyboardBacklight));
-        maybeRestoreBacklightBrightness(inputDevice, keyboardBacklight);
+		// apparently can causes a soft bootloop, so we disable it instead
+    
+        // InputDevice inputDevice = getInputDevice(deviceId);
+        // if (inputDevice == null) {
+        //     return;
+        // }
+        // final Light keyboardBacklight = getKeyboardBacklight(inputDevice);
+        // if (keyboardBacklight == null) {
+        //     mKeyboardBacklights.remove(deviceId);
+        //     return;
+        // }
+        // KeyboardBacklightState state = mKeyboardBacklights.get(deviceId);
+        // if (state != null && state.mLight.getId() == keyboardBacklight.getId()) {
+        //     return;
+        // }
+        // // The keyboard backlight was added or changed.
+        // mKeyboardBacklights.put(deviceId, new KeyboardBacklightState(deviceId, keyboardBacklight));
+        // maybeRestoreBacklightBrightness(inputDevice, keyboardBacklight);
     }
 
     private InputDevice getInputDevice(int deviceId) {
         InputManager inputManager = mContext.getSystemService(InputManager.class);
         return inputManager != null ? inputManager.getInputDevice(deviceId) : null;
-- 
2.45.2
