From 2c8bcfd8978bcd69efa0d9a9818c61ca2a90a96a Mon Sep 17 00:00:00 2001
From: kaii-lb <aosp@crave.io>
Date: Sun, 14 Jul 2024 10:22:37 +0000
Subject: [PATCH 1/3] added check for userdebug build variant, makes adb
 nonsecure

---
 core/main.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/core/main.mk b/core/main.mk
index 964a474725..7a627dc9ed 100644
--- a/core/main.mk
+++ b/core/main.mk
@@ -390,6 +390,8 @@ endif
 user_variant := user
 ifeq ($(TARGET_BUILD_VARIANT),eng)
 user_variant :=
+else ifeq ($(TARGET_BUILD_VARIANT),userdebug)
+user_variant :=
 endif
 enable_target_debugging := true
 tags_to_install :=
-- 
2.45.1
