From c22e8e815be4e962c647ac3deaf8b4c2f366705a Mon Sep 17 00:00:00 2001
From: kaii-lb <aosp@crave.io>
Date: Tue, 16 Jul 2024 12:03:11 +0000
Subject: [PATCH] disable authentication for debugging purposes

---
 daemon/main.cpp | 12 ++----------
 1 file changed, 2 insertions(+), 10 deletions(-)

diff --git a/daemon/main.cpp b/daemon/main.cpp
index e2ead883..2ea91359 100644
--- a/daemon/main.cpp
+++ b/daemon/main.cpp
@@ -198,16 +198,8 @@ int adbd_main(int server_port) {
     // descriptor will always be open.
     adbd_cloexec_auth_socket();
 
-#if defined(__ANDROID__)
-    bool device_unlocked = android::base::GetProperty("ro.boot.verifiedbootstate", "") == "orange";
-    if (device_unlocked || __android_log_is_debuggable()) {
-        // If we're on userdebug/eng or the device is unlocked, permit no-authentication.
-        auth_required = android::base::GetBoolProperty("ro.adb.secure", false);
-#if defined(__ANDROID_RECOVERY__)
-        auth_required &= android::base::GetBoolProperty("ro.adb.secure.recovery", true);
-#endif
-    }
-#endif
+    // for debugging purposes, disable authentication for ADB. remove from user builds
+    auth_required = false;
 
     // Our external storage path may be different than apps, since
     // we aren't able to bind mount after dropping root.
-- 
2.45.1
