From c6caa8055f2533c9af4330b1d6c2aa68ae43fc43 Mon Sep 17 00:00:00 2001
From: Andy CrossGate Yan <GeForce8800Ultra@gmail.com>
Date: Wed, 14 Aug 2024 12:51:05 +0000
Subject: [PATCH] andy yan's treat texfat and sdfat as exfat

Change-Id: I4c4f47c5c84a733564119e46a9452f67248e7a81
---
 Utils.cpp | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/Utils.cpp b/Utils.cpp
index d0c40ad..dc48385 100644
--- a/Utils.cpp
+++ b/Utils.cpp
@@ -1057,7 +1057,13 @@ bool IsFilesystemSupported(const std::string& fsType) {
     }
 
     /* fuse filesystems */
-    supported.append("fuse\tntfs\n");
+    supported.append("fuse\tntfs\n"
+                     "fuse\texfat\n");
+
+    /* treat sdfat/texfat as exfat */
+    if (supported.find("sdfat\n") != std::string::npos || supported.find("texfat\n") != std::string::npos) {
+        supported.append("\texfat\n");
+    }                     
 
     return supported.find(fsType + "\n") != std::string::npos;
 }
-- 
2.45.2
