From 55376b6f6a9bbd954632dbb5f01ab8556e7e50e3 Mon Sep 17 00:00:00 2001
From: kaii-lb <aosp@crave.io>
Date: Mon, 19 Aug 2024 13:31:37 +0000
Subject: [PATCH] apn patch

---
 target/product/aosp_product.mk | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/target/product/aosp_product.mk b/target/product/aosp_product.mk
index 67ab7a7..5dae7db 100644
--- a/target/product/aosp_product.mk
+++ b/target/product/aosp_product.mk
@@ -36,10 +36,3 @@ PRODUCT_PACKAGES += \
     PhotoTable \
     preinstalled-packages-platform-aosp-product.xml \
     ThemePicker \
-
-# Telephony:
-#   Provide a APN configuration to GSI product
-ifeq ($(LINEAGE_BUILD),)
-PRODUCT_COPY_FILES += \
-    device/sample/etc/apns-full-conf.xml:$(TARGET_COPY_OUT_PRODUCT)/etc/apns-conf.xml
-endif
-- 
2.45.2

