From ba1ed604301f08fef66b34cad9c21d1916fe9a76 Mon Sep 17 00:00:00 2001
From: kaii-lb <aosp@crave.io>
Date: Fri, 2 Aug 2024 08:54:17 +0000
Subject: [PATCH] removed default aosp APNs

---
 target/product/aosp_product.mk | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/target/product/aosp_product.mk b/target/product/aosp_product.mk
index 5196e6308b..becc3e5406 100644
--- a/target/product/aosp_product.mk
+++ b/target/product/aosp_product.mk
@@ -29,12 +29,5 @@ PRODUCT_PACKAGES += \
     initial-package-stopped-states-aosp.xml \
     messaging \
     PhotoTable \
     preinstalled-packages-platform-aosp-product.xml \
     ThemePicker
-
-# Telephony:
-#   Provide a APN configuration to GSI product
-ifeq ($(EVEREST_BUILD),)
-PRODUCT_COPY_FILES += \
-    device/sample/etc/apns-full-conf.xml:$(TARGET_COPY_OUT_PRODUCT)/etc/apns-conf.xml
-endif
-- 
2.45.2
