From 3723f6f327532ef68bdf2658fd7b0d6d05c9832d Mon Sep 17 00:00:00 2001
From: Alberto Ponces <ponces26@gmail.com>
Date: Wed, 7 Aug 2024 08:58:45 +0000
Subject: [PATCH] Prefer Aperture over Camera2

---
 target/product/handheld_product.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/target/product/handheld_product.mk b/target/product/handheld_product.mk
index a584e0b60f..fb49dde946 100644
--- a/target/product/handheld_product.mk
+++ b/target/product/handheld_product.mk
@@ -31,11 +31,11 @@ PRODUCT_PACKAGES += \
     SettingsIntelligence \
     frameworks-base-overlays
 
 ifeq ($(EVEREST_BUILD),)
 PRODUCT_PACKAGES += \
-    Camera2 \
+    Aperture \
     LatinIME
 endif
 
 PRODUCT_PACKAGES_DEBUG += \
     frameworks-base-overlays-debug
-- 
2.45.2
