From 4b12fe19197d43a962d38a1b8350426716be95f6 Mon Sep 17 00:00:00 2001
From: kaii-lb <aosp@crave.io>
Date: Sun, 14 Jul 2024 09:44:26 +0000
Subject: [PATCH] change ro.everest.device to EVEREST_DEVICE to match
 buildinfo.sh

---
 config/version.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/version.mk b/config/version.mk
index 3815a277..7109c094 100644
--- a/config/version.mk
+++ b/config/version.mk
@@ -94,7 +94,7 @@ PRODUCT_SYSTEM_DEFAULT_PROPERTIES += \
     ro.modversion=$(EVEREST_VERSION) \
     ro.everestos.maintainer=$(EVEREST_MAINTAINER) \
     ro.everest.edition=$(EVEREST_EDITION) \
-    ro.everest.device=$(EVEREST_BUILD)
+    ro.everest.device=$(EVEREST_DEVICE)
 
 # Signing
 ifneq (eng,$(TARGET_BUILD_VARIANT))
-- 
2.45.1
