From e3cc5d2d5a4d10d080c71a7e1b17ca2d06a05c94 Mon Sep 17 00:00:00 2001
From: kaii-;b <aosp@crave.io>
Date: Mon, 22 Jul 2024 18:27:38 +0000
Subject: [PATCH] added shared blocks and dup rules flags

---
 board-base.mk | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/board-base.mk b/board-base.mk
index 2cb5dbc..774f7ee 100644
--- a/board-base.mk
+++ b/board-base.mk
@@ -8,5 +8,11 @@ DEVICE_FRAMEWORK_MANIFEST_FILE := device/phh/treble/framework_manifest.xml
 
 BOARD_ROOT_EXTRA_FOLDERS += bt_firmware sec_storage efs persist
 BUILD_BROKEN_ELF_PREBUILT_PRODUCT_COPY_FILES := true
 
 BOARD_ROOT_EXTRA_SYMLINKS := $(filter-out $(BOARD_ROOT_EXTRA_SYMLINKS),/mnt/vendor/persist:/persist)
+
+# Build stuff
+BUILD_BROKEN_DUP_RULES := true
+
+# Compression stuff
+BOARD_EXT4_SHARE_DUP_BLOCKS := true
-- 
2.45.2
