From 6e79e2f9dd9c62b7848f639f4468b8927efb7112 Mon Sep 17 00:00:00 2001
From: kaii-lb <aosp@crave.io>
Date: Fri, 2 Aug 2024 10:09:39 +0000
Subject: [PATCH] switched to mindthegapps and removed ro.adb.secure=0

---
 generate.sh | 16 ++++++++--------
 system.prop |  1 -
 2 files changed, 8 insertions(+), 9 deletions(-)

diff --git a/generate.sh b/generate.sh
index 461eff1..6df623a 100644
--- a/generate.sh
+++ b/generate.sh
@@ -20,15 +20,20 @@ for part in a ab;do
 			for su in yes no;do
 				apps_suffix=""
 				apps_script=""
 				apps_name=""
 				extra_packages=""
-                vndk="vndk.mk"
-		optional_base=""
+				vndk="vndk.mk"
+				optional_base=""
+
+				baseArch="$arch"
+				if [ "$arch" = "a64" ];then
+					baseArch="arm"
+				fi
 				if [ "$apps" == "gapps" ];then
 					apps_suffix="g"
-					apps_script='$(call inherit-product, device/phh/treble/gapps.mk)'
+					apps_script="\$(call inherit-product, vendor/gapps/${baseArch}/${baseArch}-vendor.mk)"
 					apps_name="with GApps"
 				fi
 				if [ "$apps" == "gapps-go" ];then
 					apps_suffix="o"
 					apps_script='$(call inherit-product, device/phh/treble/gapps-go.mk)'
@@ -64,15 +69,10 @@ for part in a ab;do
 					optional_base='$(call inherit-product, device/phh/treble/base-sas.mk)'
 				fi
 
 				target="treble_${arch}_${part_suffix}${apps_suffix}${su_suffix}"
 
-				baseArch="$arch"
-				if [ "$arch" = "a64" ];then
-					baseArch="arm"
-				fi
-
 				zygote=32
 				if [ "$arch" = "arm64" ];then
 					zygote=64_32
 				fi
 
diff --git a/system.prop b/system.prop
index f3c0c45..b2bcf63 100644
--- a/system.prop
+++ b/system.prop
@@ -1,6 +1,5 @@
-ro.adb.secure=0
 #Some devices have sdcardfs kernel panicing on 8.0
 #Disable for everyone for the moment
 ro.sys.sdcardfs=0
 persist.sys.overlay.nightmode=true
 
-- 
2.45.2
