From 6e79e2f9dd9c62b7848f639f4468b8927efb7112 Mon Sep 17 00:00:00 2001
From: kaii-lb <aosp@crave.io>
Date: Fri, 2 Aug 2024 10:09:39 +0000
Subject: [PATCH] removed ro.adb.secure=0 in system.prop

---
 system.prop |  1 -
 1 file changed, 1 deletion(-)
 
diff --git a/system.prop b/system.prop
index f3c0c45..b2bcf63 100644
--- a/system.prop
+++ b/system.prop
@@ -1,6 +1,5 @@
-ro.adb.secure=0
 #Some devices have sdcardfs kernel panicing on 8.0
 #Disable for everyone for the moment
 ro.sys.sdcardfs=0
 persist.sys.overlay.nightmode=true
 
-- 
2.45.2
