From 009716a3d2e41a8f59980bb16612f5a24e306627 Mon Sep 17 00:00:00 2001
From: kaii-lb <aosp@crave.io>
Date: Mon, 22 Jul 2024 08:01:45 +0000
Subject: [PATCH] hacked around ro.everest.device dupe with diff values

---
 config/version.mk | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/config/version.mk b/config/version.mk
index 3815a277..a9f25d7f 100644
--- a/config/version.mk
+++ b/config/version.mk
@@ -79,10 +79,14 @@ EVEREST_BUILD_VERSION := $(EVEREST_NUM_VER)-$(EVEREST_CODENAME)
 EVEREST_VERSION := $(EVEREST_BUILD_VERSION)-$(EVEREST_BUILD)-$(EVEREST_BUILD_TYPE)-$(EVEREST_EDITION)-$(BUILD_TIME)-$(BUILD_DATE)
 ROM_FINGERPRINT := everest/$(PLATFORM_VERSION)/$(TARGET_PRODUCT_SHORT)/$(BUILD_TIME)
 EVEREST_DISPLAY_VERSION := $(EVEREST_VERSION)
 RELEASE_TYPE := $(EVEREST_BUILD_TYPE)
 
+# this is a hack please fix
+EVEREST_DEVICE_HOLDER := $(shell echo $(EVEREST_VERSION) | cut -f3 -d"-" | cut -f1 -d"_")
+EVEREST_DEVICE := $(shell echo tdgsi_$(EVEREST_DEVICE_HOLDER)_ab)
+
 # EverestOS System Version
 PRODUCT_SYSTEM_DEFAULT_PROPERTIES += \
     ro.everest.base.codename=$(EVEREST_CODENAME) \
     ro.everest.base.version=$(EVEREST_NUM_VER) \
     ro.everest.build.version=$(EVEREST_BUILD_VERSION) \
@@ -92,11 +96,11 @@ PRODUCT_SYSTEM_DEFAULT_PROPERTIES += \
     ro.everest.fingerprint=$(ROM_FINGERPRINT) \
     ro.everest.version=$(EVEREST_VERSION) \
     ro.modversion=$(EVEREST_VERSION) \
     ro.everestos.maintainer=$(EVEREST_MAINTAINER) \
     ro.everest.edition=$(EVEREST_EDITION) \
-    ro.everest.device=$(EVEREST_BUILD)
+    ro.everest.device=$(EVEREST_DEVICE)
 
 # Signing
 ifneq (eng,$(TARGET_BUILD_VARIANT))
 ifneq (,$(wildcard vendor/everest/signing/keys/releasekey.pk8))
 PRODUCT_DEFAULT_DEV_CERTIFICATE := vendor/everest/signing/keys/releasekey
-- 
2.45.2
