From 086e3ebc8ab512af67437d2676cfe304c0208951 Mon Sep 17 00:00:00 2001
From: "kaii_lb" <strawberry>
Date: Tues, 11 Jun 2024 19:58:26 +0000
Subject: [PATCH 1/1] remove conflict with other android.bp

---
 Android.bp | 37 -------------------------------------
 1 file changed, 37 deletions(-)

diff --git a/Android.bp b/Android.bp
index 75375be..e8a409b 100644
--- a/Android.bp
+++ b/Android.bp
@@ -452,41 +452,3 @@ cc_library_shared {
     compile_multilib: "64",
     vendor: true,
 }
-
-cc_prebuilt_library_shared {
-    name: "libprotobuf-cpp-full-vendorcompat",
-    stem: "libprotobuf-cpp-full",
-    vendor: true,
-    strip: {
-        none: true,
-    },
-    target: {
-        android_arm: {
-            srcs: ["vndk/v29/arm/libprotobuf-cpp-full.so"],
-        },
-        android_arm64: {
-            srcs: ["vndk/v29/arm64/libprotobuf-cpp-full.so"],
-        },
-    },
-    compile_multilib: "both",
-    check_elf_files: false,
-}
-
-cc_prebuilt_library_shared {
-    name: "libprotobuf-cpp-lite-vendorcompat",
-    stem: "libprotobuf-cpp-lite",
-    vendor: true,
-    strip: {
-        none: true,
-    },
-    target: {
-        android_arm: {
-            srcs: ["vndk/v29/arm/libprotobuf-cpp-lite.so"],
-        },
-        android_arm64: {
-            srcs: ["vndk/v29/arm64/libprotobuf-cpp-lite.so"],
-        },
-    },
-    compile_multilib: "both",
-    check_elf_files: false,
-}
