From 0d9f77813aa99a187e703108c309bdd6eede7b78 Mon Sep 17 00:00:00 2001
From: kaii-lb <aosp@crave.io>
Date: Fri, 12 Jul 2024 19:15:18 +0000
Subject: [PATCH] remove conflicting protobuf library

---
 Android.bp | 38 --------------------------------------
 1 file changed, 38 deletions(-)

diff --git a/Android.bp b/Android.bp
index 75375be..e8a409b 100644
--- a/Android.bp
+++ b/Android.bp
@@ -452,41 +452,3 @@ cc_library_shared {
     compile_multilib: "64",
     vendor: true,
 }
-
-cc_prebuilt_library_shared {
-    name: "libprotobuf-cpp-full-vendorcompat",
-    stem: "libprotobuf-cpp-full",
-    vendor: true,
-    strip: {
-        none: true,
-    },
-    target: {
-        android_arm: {
-            srcs: ["vndk/v29/arm/libprotobuf-cpp-full.so"],
-        },
-        android_arm64: {
-            srcs: ["vndk/v29/arm64/libprotobuf-cpp-full.so"],
-        },
-    },
-    compile_multilib: "both",
-    check_elf_files: false,
-}
-
-cc_prebuilt_library_shared {
-    name: "libprotobuf-cpp-lite-vendorcompat",
-    stem: "libprotobuf-cpp-lite",
-    vendor: true,
-    strip: {
-        none: true,
-    },
-    target: {
-        android_arm: {
-            srcs: ["vndk/v29/arm/libprotobuf-cpp-lite.so"],
-        },
-        android_arm64: {
-            srcs: ["vndk/v29/arm64/libprotobuf-cpp-lite.so"],
-        },
-    },
-    compile_multilib: "both",
-    check_elf_files: false,
-}
-- 
2.45.1
